Add clarifying note to ref two argument variant example (#889)
fishtown-analytics/docs.getdbt.com: website/docs/reference/dbt-jinja-functions/ref.md

select * from {{ ref('package_name', 'model_name') }}
```

> **Note:** The `package_name` should only include the name of the package, not the maintainer. For example, if you use the [`fivetran/stripe`](https://hub.getdbt.com/fivetran/stripe/latest/) package, you'll want to type `stripe` in that argument, and not  `fivetran/stripe`.

Reviewer: ```suggestion
**Note:** The `package_name` should only include the name of the package, not the maintainer. For example, if you use the [`fivetran/stripe`](https://hub.getdbt.com/fivetran/stripe/latest/) package, type `stripe` in that argument, and not  `fivetran/stripe`.
```

select * from {{ ref('package_name', 'model_name') }}
```

> **Note:** The `package_name` should only include the name of the package, not the maintainer. For example, if using the `fivetran/stripe` package, you'll want to put `stripe` in that argument, rather than `fivetran/stripe`.

Reviewer: ```suggestion
**Note:** The `package_name` should only include the name of the package, not the maintainer. For example, if you use the [`fivetran/stripe`](https://hub.getdbt.com/fivetran/stripe/latest/) package, you'll want to type `stripe` in that argument, and not  `fivetran/stripe`.

`select * from {{ ref('stripe', 'dim_orders') }}`
```
